### PR TITLE
Fixes the plugin crash when version field is not available.

### PR DESCRIPTION
**Pull request by mashhurs**
### Description
We got following error message indicates that version field may be null so that leads plugin crash.

```
{"level":"ERROR","loggerName":"logstash.codecs.cef","timeMillis":1721266426115,"thread":"nioEventLoopGroup-8-1","logEvent":{"message":"Failed to decode CEF payload. Generating failure event with payload in message field.","exception":{"metaClass":{"metaClass":{"exception":"NoMethodError","message":"undefined method `include?' for nil:NilClass","backtrace":["/usr/share/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-codec-cef-6.2.5-java/lib/logstash/codecs/cef.rb:241:in `handle'","/usr/share/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-codec-cef-6.2.5-java/lib/logstash/codecs/cef.rb:197:in `block in decode'","org/jruby/RubyArray.java:1821:in `each'","/usr/share/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-codec-cef-6.2.5-java/lib/logstash/codecs/cef.rb:196:in `decode'","/usr/share/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-input-tcp-6.3.0-java/lib/logstash/inputs/tcp.rb:194:in `decode_buffer'","/usr/share/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-input-tcp-6.3.0-java/lib/logstash/inputs/tcp/decoder_impl.rb:22:in `decode'"],"original_data":"1245"}}}}}
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 6.2.8
+ - Fixes the plugin crash when version field is not available [#104](https://github.com/logstash-plugins/logstash-codec-cef/pull/104)
```
Comment (andsel) [inline CHANGELOG.md:2]: I would suggest to use the other changelogs format:
```suggestion
 - Fix: avoid to crash when version field is not available [#104](https://github.com/logstash-plugins/logstash-codec-cef/pull/104)
```

File: lib/logstash/codecs/cef.rb [hunk 2]
```diff
@@ -271,7 +271,8 @@ def handle(data, &block)
 
     # Try and parse out the syslog header if there is one
     cef_version_field = @header_fields[0]
-    if (cef_version = event.get(cef_version_field)).include?(' ')
+    cef_version = event.get(cef_version_field)
+    if cef_version && cef_version.include?(' ')
```
Comment (andsel) [inline lib/logstash/codecs/cef.rb:275]: Maybe worth to cover the case with a test

**Review by andsel — CHANGES_REQUESTED** [review]
Left a couple of comments on standardising changelog message and cover the change with a test.

**Comment by mashhurs** [comment]
Covered by https://github.com/logstash-plugins/logstash-codec-cef/pull/98

**Pull request closed without merging**